api/src/organization/organization.ts

```diff
@@ -100,6 +99,17 @@ async function ensureOrganizationAddress(
   return organizationAddress;
 }
 
+export async function organizationExists(
+  multichain: MultichainClient,
+  organization: Organization,
+): Promise<boolean> {
+  if (await getOrganizationAddressItem(multichain, organization)) {
```

Comment:
someone who is not familiar with the codebase might be confused by this line. please add a descriptive comment to this line (and maybe put it into a separate variable, e.g. `const organizationHasItems = await getOrganizationAddressItem(multichain, organization) ? true : false` and then use this as condition